spec/text_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'text'
+
+describe Text do
+
+		it 'sends a confirmation text' do
+				expect { subject.send_sms }.not_to raise_error
```

Comment:
Try to find a way to check that your text object interacts with the Twilio object in the way you expect it to, either by stubbing the methods called on the Twilio object, or by passing in a double of the Twilio object.